packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2764,6 +2764,7 @@ export interface ImageGalleryImageProps {
 // @beta
 export interface ImageGalleryProps {
     images: Array<ImageGalleryImageProps>;
+    isOpen?: boolean;
```

Comment:
`isOpen` sounds a bit odd to me, maybe `isVisible` or `isShown` or `isActive` or `isOpened`?

Reply:
> `isOpen` sounds a bit odd to me, maybe `isVisible` or `isShown` or `isActive` or `isOpened`?

I think we briefly talked about this in the last internal review and we tend to agree to follow the naming of the FluentUI Modal component which is isOpen. @JoshuaLai Do you remember whether my statement is correct?

Reply:
Yes. If isOpen is what FluentUI uses lets try to match their naming